bf2fc6cc711aee1a0c2a/kas-fleet-manager
# chore: deprecate the /status endpoint

## Description
The /status endpoint only reports whether the overall maximum capacity has been reached. This no longer makes sense as we now support multiple regions and Kafka instance types. Instead, capacity should  be reported per region and instance type. This information will be available in the GET /api/kafkas_mgmt/v1/cloud_providers/{id}/regions endpoint resulting in the /status endpoint to become obsolete.

The /status endpoint should be deprecated to allow time for clients to switch to using the cloud provider regions endpoint instead.

**NOTE**: Please do not merge until changes to add the capacity information in the cloud providers regions endpoint has been merged. 

## Verification Steps
1. Run make run/docs
    - Ensure that the /status endpoint is marked as deprecated. 
    - Ensure the /status endpoint description makes sense.

## Checklist (Definition of Done)
